samples/client/petstore/dart-jaguar/openapi/lib/model/order.dart

```diff
@@ -5,22 +5,24 @@ part 'order.jser.dart';
 
 class Order {
   
-  @Alias('id', isNullable: false)
+  @Alias('id', isNullable: false,  )
   final int id;
   
-  @Alias('petId', isNullable: false)
+  @Alias('petId', isNullable: false,  )
   final int petId;
   
-  @Alias('quantity', isNullable: false)
+  @Alias('quantity', isNullable: false,  )
   final int quantity;
   
-  @Alias('shipDate', isNullable: false)
+  @Alias('shipDate', isNullable: false,  )
   final DateTime shipDate;
    /* Order Status */
-  @Alias('status', isNullable: false)
+  @Alias('status', isNullable: false,
+             processor:  const StringFieldProcessor(),
```

Comment:
@wing328 is there a way to know if the enum is from and #ref or inline ? Because this will not work :/ 
#ref enum are generating a new file and class, but inline are not for some reason and does this instead

Reply:
I have ignored inline enum and handle them as simple string, I don't think their is another easy way to do that for now, at least the compiled code works like this